content/general/concepts/unicode/unicode.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Unicode'
+Description: 'Unicode is a text-processing standard that assigns a unique number to each character across most human languages and writing systems.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Characters'
+  - 'Symbol'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+**Unicode** is a text-processing standard that assigns a unique number to each character across most human languages and writing systems. It is device- and platform-agnostic and is maintained by the [Unicode Consortium](https://home.unicode.org/). The adoption of the Unicode standard has grown so much that it is now implemented in many modern operating systems, programming languages, as well as [XML](https://www.codecademy.com/resources/docs/general/xml).
+
+## Common Character Sets
+
+Below are some tables of common Unicode character sets.
+
+> **Note:** These tables only showcase a few characters from each set. More information can be found in [the official Unicode character code charts](https://www.unicode.org/charts).
```

Comment:
Just making sure we wanted this as a quote

Reply:
Yeah, this is something new that we're trying out to better differentiate a **Note** from the normal flow of the rest of the entry. The Markdown blockquote is sort of a work-around to achieve something similar to Google's [precautionary notices](https://developers.google.com/style/notices).

Reply:
Works for me!